portfolio/src/main/webapp/script.js

```diff
@@ -65,17 +65,21 @@ function loadComments(){
 }
 
 window.onload = (event) => {
-  if(params.get("?loggedIn") == "0"){
-    const displayComments = document.getElementById("display-comments");
-    removeChildren(displayComments);
-    const message = document.createElement("a");
-    message.setAttribute("href",params.get("url"));
-    message.innerText = "Click here to Log in and see/add comments!";
-    displayComments.appendChild(message);
-  }
-  else{
-    const logoutUrl = document.getElementById("logout-url");
-    logoutUrl.setAttribute("href",params.get("url"));
-    logoutUrl.innerText = "LOGOUT";
-  }
+  fetch(`validate-email?email=${params.get("?email")}&token=${params.get("token")}`).then(response => response.text()).then((check) =>{
+    console.log(check);
+    if(check == 0 || check == "0"){
+      console.log(check);
+      const displayComments = document.getElementById("display-comments");
+      removeChildren(displayComments);
+      const message = document.createElement("a");
+      message.setAttribute("href",params.get("url"));
+      message.innerText = "Click here to Log in and see/add comments!";
+      displayComments.appendChild(message);
+    }
+    else{
```

Comment:
Make this newline consistent

```
if () {
} else {
}
```

Reply:
Each line is 2 spaces over, could you clarify please